snowfall/training/mmi_mbr_graph.py

```diff
@@ -146,10 +147,11 @@ def compile(self, texts: Iterable[str],
 
         num_graphs_with_self_loops = k2.arc_sort(num_graphs_with_self_loops)
 
+        # inherit the `phones` attribute from ctc_topo_P
         num = k2.compose(ctc_topo_P,
                          num_graphs_with_self_loops,
-                         treat_epsilons_specially=False,
-                         inner_labels='phones')
+                         treat_epsilons_specially=False)
```

Comment:
Removing `inner_labels` here so that `num` inherits the `phones` attribute from
`ctc_topo_P`. But it does not affect the result since it uses only `mbr_lats` and `den_lats`.